### PR TITLE
Tack on client-side useragent for simpleUA

### PR DESCRIPTION
**Pull request by schwers**
:eyeglasses: @ajacksified

### CODE REVIEW

File: src/lib/errorLog.es6.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ function simpleUA(agent) {
   if (/server/i.test(agent)) { return 'server'; }
   if (/iPhone/i.test(agent)) { return 'ios'; }
   if (/android/i.test(agent)) { return 'android'; }
+  if (/Windows Phone/i.test(agent) || /Trident/i.test(agent)) { return 'windows-phone'; }
+  if (/googlebot/i.test(agent)) { return 'googlebog-jsclient'; }
```
Comment (ajacksified) [inline src/lib/errorLog.es6.js]: > googlebog

**Comment by ajacksified** [comment]
:no_bell: Can we distinguish old android vs android chrome, and same for ios? (safari vs chrome)

:haircut:

**Comment by ajacksified** [comment]
err, I mean :nail_care:

**Comment by schwers** [comment]
:haircut:

File: src/lib/errorLog.es6.js [hunk 2]
```diff
@@ -2,8 +2,27 @@ import superagent from 'superagent';
 
 function simpleUA(agent) {
   if (/server/i.test(agent)) { return 'server'; }
-  if (/iPhone/i.test(agent)) { return 'ios'; }
-  if (/android/i.test(agent)) { return 'android'; }
+
+  if (/iPhone/i.test(agent) || /iPad/i.test(agent) || /iPod/i.test(agent)) {
+    if (/CriOS/i.test(agent)) {
+      return 'ios-chrome';
+    } else {
```
Comment (ajacksified) [inline src/lib/errorLog.es6.js]: no `else`, since the `if` returns (new linter gets angry)

File: src/lib/errorLog.es6.js [hunk 3]
```diff
@@ -2,8 +2,27 @@ import superagent from 'superagent';
...
+    }
+  }
+
+  // Windows Phone 10 adds android to the UA, put this test first
+  if (/Windows Phone/i.test(agent) || /Trident/i.test(agent)) { return 'windows-phone'; }
+
+  if (/android/i.test(agent)) {
+    if (/Version/i.test(agent)) {
+      return 'android-stock-browser';
+    } else {
```
Comment (ajacksified) [inline src/lib/errorLog.es6.js]: no `else`, since the `if` returns (new linter gets angry). just return

**Comment by ajacksified** [comment]
:nail_care:

**Comment by schwers** [comment]
:haircut:

**Comment by curioussavage** [comment]
:+1:

**Comment by ajacksified** [comment]
:fish:

**Pull request merged**